PokeAlarm/FacebookPage/FacebookPageAlarm.py

```diff
@@ -70,16 +82,29 @@ def startup_message(self):
     def create_alert_settings(self, settings, default):
         alert = {
             'message': settings.pop('message', default['message']),
-            'link': settings.pop('link', default['link'])
+            'link': settings.pop('link', default['link']),
+            'caption': settings.pop('caption', default['caption']),
+            'description': settings.pop('description', default['description']),
+            'image': settings.pop('image', default['image']),
+            'name': settings.pop('name', default['name'])
         }
         reject_leftover_parameters(settings, "'Alert level in FacebookPage alarm.")
         return alert
 
     # Post Pokemon Message
     def send_alert(self, alert, info):
+        attachment = {"link": replace(alert['link'], info)}
+        if alert['caption']:
+            attachment['caption'] = replace(alert['caption'], info)
+        if alert['description']:
```

Comment:
I think I missed this before, but can any of these but caption actually be unset?